Sources/LightweightCharts/LightweightChartsModels/Options/Basics/TimeScaleOptions.swift

```diff
@@ -44,7 +44,7 @@ public struct TimeScaleOptions {
         self.barSpacing = barSpacing
         self.minBarSpacing = minBarSpacing
         self.fixLeftEdge = fixLeftEdge
-        self.fixRightEdge = fixLeftEdge
+        self.fixRightEdge = fixRightEdge
```

Comment:
Guess we also missed to add this field to CodingKeys

Reply:
af596e1e9d62738b79533a2ab88f143e374fc872